superset/db_engine_specs/mssql.py

```diff
@@ -48,6 +48,9 @@ class MssqlEngineSpec(BaseEngineSpec):
     limit_method = LimitMethod.WRAP_SQL
     max_column_name_length = 128
     allows_cte_in_subquery = False
+    allow_limit_clause = False
+    sel_keywords = {"SELECT"}
+    top_keywords = {"TOP"}
```

Comment:
wasn't this called `select_keywords` in `BaseEngineSpec`? Also, no need to override if the property is unchanged in the extending class.

Reply:
Corrected this.